docs/Deep-Dives/metaapi.md

```diff
@@ -0,0 +1,200 @@
+title: Flotiq and Meta-API integration | Flotiq docs
+description: This tutorial shows how to use Flotiq and Meta-API to integrate with another API
+
+
+# Flotiq + Meta-API + Google Spreadsheets use case
+
+This guide shows how to add your private Flotiq API to Meta-API and prepare an example Spell. 
+In our case we will create a report in Google Spreadsheet based on products stock quantity.
+
+![Integrate Flotiq with Google Spreadsheets using Meta-API Spell](images/metaapi/img-0.png){: .border}
+
+
+## 1. Create an empty sheet in Google Spreadsheets
+
+Let's prepare our spreadsheet to use in further steps. 
+We add two columns (Date, Stock Quantity) and a chart with the default config for these columns.
+
+![Create an empty sheet to store Flotiq aggregated data](images/metaapi/meta-spr-1.png){: .border}
+
+Notice the `spreadsheet id`. 
+It's located in the URL `https://docs.google.com/spreadsheets/d/[___here___]/edit#gid=0`. 
+We will use it in the next steps.
+
+
+## 2. Model your data in Flotiq
+
+Sign in to your Flotiq account and make sure that you have added your Content Type Definitions. 
+In our example, we use [predefined Product CTD](https://flotiq.com/docs/panel/predefined-content-types/) with an extra property named `stock_quantity`.
+
+!!! Note
+    If you are new to Flotiq, check the [Dashboard > Content Types](https://flotiq.com/docs/panel/content-types/)
+    section in the docs
+
+![Product Content Type Definition in Flotiq](images/metaapi/meta-flotiq-1.png){: .border}
+
+Now it's time to add some example data. 
+As we are creative, we added more sophisticated data than "Lorem ipsum" :)
```

Comment:
I don't really understand this part, when did we add data? 
Are we going to add it in the next step?
Is the data already added?